toolchainlib/pesde.toml

```diff
@@ -6,21 +6,21 @@ repository = "https://github.com/pesde-pkg/tooling/tree/main/toolchainlib"
 includes = ["src/**/*.luau", "pesde.toml", "LICENSE", "README.md"]
 
 [engines]
-lune = "^0.8.7"
-pesde = "^0.6.0"
+lune = "^0.10.2"
+pesde = "^0.7.0"
 
 [target]
 environment = "lune"
-lib = "src/init.luau"
+lib = "src/lib.luau"

```

Comment:
Why did we rename it to `lib.luau`? Feels less idiomatic.

A consumer shouldn't have to require `./lune_packages/toolchainlib/lib`!

Reply:
That's weird. Please consider fixing this.